business-registry-dashboard/tests/unit/utils/business.test.ts

```diff
@@ -217,7 +217,7 @@ describe('business utils', () => {
 
       expect(_fetch).toHaveBeenCalledWith(expect.stringContaining('/businesses?draft=true'), expect.objectContaining({
         method: 'POST',
-        headers: { Authorization: 'Bearer 123' },
+        headers: { 'App-Name': 'Business Registry Dashboard', Authorization: 'Bearer 123' },
```

Comment:
This isn't needed on line 254 because that uses `toMatchObject` while this line uses `objectContaining`.